src/up.cmd.js

```diff
@@ -86,6 +86,11 @@ export default class UpCommand extends AbstractServerCommand {
     }
     if (!this._url) {
       await this.verifyUrl(this._gitUrl, ref);
+    } else if (/\{\{(owner|repo)+\}\}/.test(this._url)) {
+      const { owner, repo } = await GitUtils.getOriginURL(this.directory);
```

Comment:
why didn't you reuse `this._gitUrl` ?

```suggestion
      const { owner, repo } = this._gitUrl;
```